client/src/util.ts

```diff
@@ -12,12 +15,7 @@ export function assert(cond: unknown, msg = "Assertion failed."): asserts cond {
   }
 }
 
-let memoizedCommand: string | undefined;
```

Comment:
I don't think we should memoize this as it might prevent us from getting the latest information (ex. `"deno.path"` might have changed). This is only used when starting the language service anyway.